packages/suite/src/views/wallet/send/components/Options/components/BitcoinOptions/index.tsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import styled from 'styled-components';
+import { darken } from 'polished';
 import { Translation } from '@suite-components';
 import { OnOffSwitcher } from '@wallet-components';
 import { Button, Tooltip } from '@trezor/components';
 import { useSendFormContext } from '@wallet-hooks';
 import { isEnabled as isFeatureEnabled } from '@suite-utils/features';
+import { useActions } from '@suite-hooks';
+import * as guideActions from '@suite-actions/guideActions';
+import { Node } from '@trezor/suite-data/src/guide/parser';
```

Comment:
```suggestion
import type { Node } from '@suite-types/guide';
```

Guide types from parser are reexported in `guide.tsx` file. (Note: We should change `guide.tsx` to `guide.ts`)